beaker/src/model_cache.rs

```diff
@@ -127,9 +280,20 @@ mod tests {
 
     #[test]
     fn test_cache_dir() {
+        // Test without ONNX_MODEL_CACHE_DIR set
+        std::env::remove_var("ONNX_MODEL_CACHE_DIR");
```

Comment:
With this effect later test? Seems we’d want to restore the existing value after